Plots/Panels/NCL_panel_4.py

```diff
@@ -0,0 +1,165 @@
+"""
+NCL_panel_4.py
+===============
+This script illustrates the following concepts:
+   - Paneling three plots vertically on a page
+   - Adding a common title to paneled plots
+   - Adding a common labelbar to paneled plots
+   - Adding additional text at the bottom of a series of paneled plots
+   - Subsetting a color map
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/panel_4.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/panel_4_lg.png
+"""
+
+###############################################################################
+# Import packages:
+
+import cartopy.crs as ccrs
+from cartopy.mpl.gridliner import LongitudeFormatter, LatitudeFormatter
+import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1.inset_locator import inset_axes
+import numpy as np
+import xarray as xr
+
+import geocat.datafiles as gdf
+from geocat.viz import cmaps as gvcmaps
+import geocat.viz.util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into
+# xarrays, choosing the 2nd timestamp
+ds = xr.open_dataset(gdf.get("netcdf_files/uv300.nc")).isel(time=1)
+
+###############################################################################
+# Utility Function: Labelled Filled Contour Plot:
+
+# Define a utility plotting function in order not to repeat many lines of codes
+# since we need to make the same figure with two different variables.
+
+
+def plot_labelled_filled_contours(data, ax=None):
+    """A utility function for convenience that plots labelled, filled contours
+    with black contours marking each level.It will return a dictionary
+    containing three objects corresponding to the filled contours, the black
+    contours, and the contour labels."""
```

Comment:
```suggestion
    contour outlines, and the contour labels."""
```
Can also be "contour levels" instead of outlines